judgments/templatetags/court_utils.py

```diff
@@ -53,3 +62,36 @@ def get_court_date_range(court):
         return str(start_year)
     else:
         return mark_safe("%s&nbsp;to&nbsp;%s" % (start_year, end_year))
+
+
+@register.filter
+def get_court_judgments_count(court):
+    return search_judgments_and_parse_response(
+        api_client, SearchParameters(court=court.canonical_param)
+    ).total
+
+
+@register.filter
+def get_court_intro_text(court):
+    def read_markdown(param):
```

Comment:
I think a happy-case test for markdown feels valuable.

Reply:
aha i added some tests on another PR, this one got merged before i could sneak them in. I should've done them before opening the PR really.